players/Randomplayer.py

```diff
@@ -0,0 +1,49 @@
+#!/usr/bin/env python3
+
+
+from .Player import Player
+import random
+
+
+class Randomplayer(Player):
```

Comment:
Usually we use TitleCase for python classes, e.g. DataFrame, CoinFlip

Reply:
Noted and updated!